src/views/PackageDetail.vue

```diff
@@ -195,6 +195,132 @@
+                <div v-if="importTreeAnalysis.tree">
+                  <div>
+                    <input
+                      class="collapse-switch"
+                      type="checkbox"
+                      id="collapse-switch-1"
+                    />
+                    <label class="panel-block" for="collapse-switch-1">
+                      <font-awesome-icon
+                        class="icon-margin-right"
+                        :icon="['fa', 'external-link-alt']"
+                      />
+                      <span
+                        class="collapse-arrow"
+                        v-if="importTreeAnalysis.dependencies.length"
+                        >{{ importTreeAnalysis.dependencies.length }} external
+                        dependencies
+                      </span>
+                      <span v-else>No external dependencies 🎉</span>
+                    </label>
+                    <div
+                      class="panel-block panel-list collapse"
+                      v-if="importTreeAnalysis.dependencies.length"
+                    >
+                      <ul>
+                        <li
+                          class="panel-list-simple"
+                          v-for="(url,
+                          index) in importTreeAnalysis.dependencies"
+                          :key="`panel-dependencies-${index}`"
+                        >
+                          <UrlRegistry :url="url" :raw="rawUrls"></UrlRegistry>
+                        </li>
+                      </ul>
+                    </div>
+                  </div>
+                  <div>
+                    <input
+                      class="collapse-switch"
+                      type="checkbox"
+                      id="collapse-switch-2"
+                    />
+                    <label class="panel-block" for="collapse-switch-2">
+                      <font-awesome-icon
+                        class="icon-margin-right"
+                        :icon="['fa', 'file-import']"
+                      />
+                      <span
+                        class="collapse-arrow"
+                        v-if="importTreeAnalysis.count"
+                        >{{ importTreeAnalysis.count }} imports
+                      </span>
+                      <span v-else>No imports 🎉</span>
+                    </label>
+                    <div
+                      class="panel-block panel-list collapse"
+                      v-if="importTreeAnalysis.count"
+                    >
+                      <ul>
+                        <li
+                          class="panel-list-simple"
+                          v-for="(url, index) in importTreeAnalysis.imports"
+                          :key="`panel-imports-${index}`"
+                        >
+                          <UrlRegistry :url="url" :raw="rawUrls"></UrlRegistry>
+                        </li>
+                      </ul>
+                    </div>
```

Comment:
You could set a fixed height for this and an `overflow-y: auto`. That way, users won't have to scroll up if they want to read the `README` again, and only the actual tree would be scrolled

Reply:
What height do you recommend?

Reply:
Not sure. Try 10em or 12em

Reply:
```css
.panel-list
  overflow-x: auto
  max-height: 12em
  overflow-y: auto
```
a bit too small imo
![image](https://user-images.githubusercontent.com/52361520/95614948-b0db3f80-0a67-11eb-84dc-3453e6eb29de.png)
I'm going to need your help, it's breaking while doing this. I can't access the root of the tree anymore.

Reply:
One thing that has gone through my mind before was fixing the position of the sidebar so when a user scrolls, it's still available. Just an idea. That would also give more room for that component. Unfortunately, I am unsure exactly how to fix that specific issue.